### PR TITLE
feat: increase spacing to bottom for sidebar button

### PR DESCRIPTION
**Pull request by kesselb**
| Before | After |
|--------|--------|
| Screenshot from 2023-08-04 18-35-47 | Screenshot from 2023-08-04 18-36-23 |

### CODE REVIEW

File: src/views/ProfilerToolbar.vue [hunk 1]
```diff
@@ -413,4 +413,8 @@ export default {
 	// overwrite body height for profiler toolbar
 	--body-height: calc(100% - env(safe-area-inset-bottom) - 50px - var(--body-container-margin) - 36px);
 }
+.app-sidebar-tab__buttons {
```
Comment (susnux) [inline src/views/ProfilerToolbar.vue:416]:
```suggestion

.app-sidebar-tab__buttons {
```

**Review by susnux — APPROVED** [review]
Just one missing new line for stylelint

File: src/views/ProfilerToolbar.vue (hunk 1 above)
Comment (kesselb) [reply src/views/ProfilerToolbar.vue:416]: :see_no_evil:

Comment (kesselb) [reply src/views/ProfilerToolbar.vue:417]: `npm run stylelint:fix` did the trick :tada:

**Pull request merged**